lib/recurly/errors.js

```diff
@@ -39,8 +39,22 @@ const GOOGLE_PAY_ERRORS = [
     classification: 'internal'
   },
 ];
+const PAYMENT_METHODS_ERRORS = [
```

Comment:
I agree, this generic name could be really confused because credit cards are payment methods and they are not here. Let's talk about it in eng hangout.